BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -71,6 +71,49 @@
     single_file = <%= single_file ? "\"#{single_file['id']}\"" : "nil" %>
 %%>
 
+
```

Comment:
Isn't this duplicating completely the lambdas that are already defined at line 508 (original code, line 558 in the new code) ?